src/main/java/at/hannibal2/skyhanni/features/misc/SkyBlockKickDuration.kt

```diff
@@ -49,10 +49,10 @@ object SkyBlockKickDuration {
 
     @HandleEvent
     fun onChat(event: SkyHanniChatEvent) {
-        if (!isEnabled()) return
+        if (!isEnabled() && !showTime) return
```

Comment:
```suggestion
        if (!isEnabled() || !showTime) return
```
i believe this should be an or